op-acceptance-tests/tests/isthmus/withdrawal_root/withdrawals_root_test.go

```diff
@@ -0,0 +1,42 @@
+package withdrawal
+
+import (
+	"testing"
+
+	"github.com/ethereum-optimism/optimism/op-devstack/devtest"
+	"github.com/ethereum-optimism/optimism/op-devstack/dsl"
+	"github.com/ethereum-optimism/optimism/op-devstack/presets"
+	"github.com/ethereum-optimism/optimism/op-node/rollup"
+	"github.com/ethereum-optimism/optimism/op-service/eth"
+)
+
+func TestWithdrawalRoot(gt *testing.T) {
+	t := devtest.SerialT(gt)
```

Comment:
nit: does this actually need to a be serial or could it run in parallel with other tests?